transform/models/marts/fact_stars_monthly.sql

```diff
@@ -0,0 +1,30 @@
+with monthly_stars as (
+  select 
+    substr(s.date,5,2) as date_month, 
+    s.repo_id,
+    count(s.count) as count_stars
+  from {{ ref('fact_stars') }} as s
+  group by 1, 2
+),
+
+monthly_stars_with_lag as (
+  select
+    ms.date_month,
+    ms.repo_id,
+    ms.count_stars,
+    lag(ms.count_stars) over (
+      partition by ms.repo_id 
+      order by ms.date_month
+      rows between 12 preceding and 12 preceding
```

Comment:
Nice that you used the preceding notation! While this works, a more thematic way would be to use the second arg of lag to determine how many rows to look back to: `lag(ms.count_stars, 12)`